.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,208 @@
+name: Integration Tests
+
+on:
+  pull_request: { }
+  push:
+    branches: [ main, develop ]
+
+# Enrich gradle.properties for CI/CD
+env:
+  CI_GRADLE_ARG_PROPERTIES: >
+    -Porg.gradle.jvmargs=-Xmx2g
+    -Porg.gradle.parallel=false
+
+jobs:
+  # Build Android Tests [Matrix SDK]
+  build-android-test-matrix-sdk:
+    name: Matrix SDK - Build Android Tests
```

Comment:
Do we want to keep this action since it is now done when we run the integration test?
It has been added (see the comment [here](https://github.com/vector-im/element-android/pull/4842/files#diff-82454b2fc887e9985b9348b8f3bca03d2f839645a4c3ba1f3f850014d7da5c8bL16)) to ensure that the tests are compiling by making this action "Required" on every PRs.
But now I think we can put the test run "Required".

Thinking more about it, since this action is more reliable than the running of integration test, maybe we can keep it.

Reply:
Exactly thats why I kept it. While its not 100% reliable I think we should keep it